src/scan_nut.cc

```diff
@@ -134,6 +134,71 @@ s_nut_dumpdata_to_fty_message (fty_proto_t *fmsg, std::map <std::string, std::st
     }
 }
 
+void
+s_nut_dumpdata_daisychain_to_fty_message (fty_proto_t *asset, std::map <std::string, std::string> &dump, zsock_t* pipe)
+{
+    if(! asset) return;
+
+    auto item = dump.find ("device.count");
+    if(item == dump.end () || streq(item->second.c_str(), "1")) {
+        s_nut_dumpdata_to_fty_message(asset, dump);
+        return;
+    }
+    int nbDevice = std::stoi(item->second);
+
+    for(int i= 1; i <= nbDevice; i++) {
+        std::string first_part = "device." + std::to_string(i);
+        static std::map <std::string, std::string> mapping = {
+            {first_part + ".model", "model"},
+            {first_part + ".ups.model", "model"},
+            {first_part + ".mfr", "manufacturer"},
+            {first_part + ".ups.mfr", "manufacturer"},
+            {first_part + ".serial", "serial_no"},
+            {first_part + ".description", "device.description"},
+            {"device.contact", "device.contact"},
+            {"device.location", "device.location"},
+            {first_part + ".part", "device.part"},
+            {first_part + ".ups.serial", "serial_no"},
+            {first_part + ".ups.firmware", "firmware"},
+            {first_part + ".input.phases", "phases.input"},
+            {first_part + ".outlet.count", "outlet.count"}
+        };
+
+        fty_proto_t *fmsg;
+        if(i != 1) {
+            fmsg = fty_proto_new(FTY_PROTO_ASSET);
+
+            fty_proto_ext_insert (fmsg, "ip.1", "%s", fty_proto_ext_string(asset,"ip.1", ""));
```

Comment:
@aquette : do we (already, or expect to) support daisy-chain over non-networked connections, e.g. via USB/serial connection to the UPS (so driver is likely set up via `upsconf_block`) and should it matter, if such asset is stored with a formal IP of `127.0.0.1` anyhow?